Gemfile

```diff
@@ -42,6 +42,7 @@ gem 'simple_form'
 gem 'sinatra', require: false
 gem 'slim-rails'
 gem 'split', require: 'split/dashboard'
+gem 'stringex'
```

Comment:
looks like the stringex gem just wraps the unidecoder gem, which provides `String#to_ascii`, should we just use that instead?